package.json

```diff
@@ -28,6 +28,7 @@
     "clampify": "^1.2.1",
     "datebook": "^2.0.3",
     "font-awesome": "^4.7.0",
+    "global": "^4.4.0",
```

Comment:
wait what are we using global for, I don't see any reference to it in the codebase